docs/sphinx_source/kingmaker.rst

```diff
@@ -144,18 +161,18 @@ To trigger a production of ntuples run
+    law run ProduceSamples --analysis template_analysis --config template_config --production-tag test_production_v1 --sample-list samples.txt --scopes mm --shifts None --workers 8 
 
 
 The different options are:
 
 - ``--analysis``: The analysis to be used. The name corresponds to the analysis folder in the ``CROWN/analysis_configurations`` folder.
-- ``--config``: The config file to be used. The config file contains the information about the samples, the input files, the output files, the friend trees, the branches to be read, etc. The config file is located in the ``CROWN/analysis_configurations/<analysis>/config`` folder.
+- ``--config``: The config file to be used. The config file contains the information about all producers, parameters and output branches that are needed to run the ntuple production of your analysis. The config file is located in the ``CROWN/analysis_configurations/<analysis>/`` folder.
+- ``--production-tag``: The production tag is used to identify the production. It is used to create the output folder and the output files. The output files are stored in the ``/<base>/<production-tag>/CROWNRun/`` folder. The ``base`` variable is set using the Configuration. By default, it is set to ``root://cmsxrootd-kit-disk.gridka.de//store/user/${USER}/CROWN/ntuples/``. Within the ``CROWNRun`` folder, the different samples are stored, matching the ``<era>/<samplenick>/<channel>/<samplenick>_<counter>.root`` pattern.
 - ``--sample-list``: The sample list to be used. The sample list can be generated by the ``sample_manager`` and contains the information about the samples to be processed. The sample nicks can also be provided as a comma-separated list.
-- ``--production-tag``: The production tag is used to identify the production. It is used to create the output folder and the output files. The output files are stored in the ``/<base>/<production-tag>/CROWNRun/`` folder. The ``base`` variable is set using the Configuration. By default, it is set to ``root://cmsxrootd-kit-disk.gridka.de//store/user/${USER}/CROWN/ntuples/``. Within the folder, the different samples are stored, matching the ``<era>/<samplenick>/<channel>/<samplenick>_<counter>.root`` pattern.
-- ``--workers``: The number of workers to be used. Each worker is responsible for the submission and handling of one sample. The number of workers should be at least the number of samples.
 - ``--scopes``: The scopes to be used, provided as a comma-separated list.
-- ``--shifts``: The shifts to be used, provided as a comma-separated list. If no shifts are provided, no shifts are applied. If ``All`` is provided, all shifts are applied, if ``None`` is provided, no shifts are applied.
+- ``--shifts``: The shifts to be used, provided as a comma-separated list. If this parameter is not set, no shifts are applied. If ``All`` is provided, all shifts are applied, if ``None`` is provided, no shifts are applied.
```

Comment:
Maybe out of scope for this PR, but I believe the defaults for whether shifts are used or not is different between KingMaker and CROWN. At some point this should be unified.

Reply:
I changed the default in CROWN to `none`.